cmake/modules/FindHarfBuzz.cmake

```diff
@@ -0,0 +1,46 @@
+#.rst:
+# FindHarfbuzz
+# ------------
+# Finds the HarfBuzz library
+#
+# This will define the following variables::
+#
+# HARFBUZZ_FOUND - system has HarfBuzz
+# HARFBUZZ_INCLUDE_DIRS - the HarfBuzz include directory
+# HARFBUZZ_LIBRARIES - the HarfBuzz libraries
+#
+# and the following imported targets::
+#
+#   HarfBuzz::HarfBuzz   - The HarfBuzz library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_HARFBUZZ harfbuzz QUIET)
+endif()
+
+find_path(HARFBUZZ_INCLUDE_DIR NAMES harfbuzz/hb-ft.h hb-ft.h
+                               PATHS ${PC_HARFBUZZ_INCLUDEDIR}
+                                     ${PC_HARFBUZZ_INCLUDE_DIRS}
```

Comment:
not needed

Reply:
> not needed

removed.

Reply:
@wsnipex This is need right? Im getting 
`Could NOT find HarfBuzz (missing: HARFBUZZ_INCLUDE_DIR) (found version
  "2.8.0")
`
while building for android without this line. I dont know anything about makefiles. This was modified from `FindFreeType.cmake`

Reply:
I get same error on LibreELEC. Seems find_path must be set just like for freetype.